src/controllers/profileController.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const bcrypt = require('bcrypt');
+const userService = require('../services/userService.js');
+const emailService = require('../services/emailService.js');
+const { ApiError } = require('../exceptions/ApiError.js');
+
+async function getUserById(userId) {
```

Comment:
The function getUserById is not exported, but it could be useful in other modules as well. Consider exporting it if it's intended to be used elsewhere.